src/main/scala/Vcs.scala

```diff
@@ -136,7 +136,12 @@ class Git(val baseDir: File) extends Vcs with GitLike {
     cmd("push", trackingRemote, "%s:%s" format (localBranch, trackingBranch))
   }
 
-  private def pushTags = cmd("push", "--tags", trackingRemote)
+  private def pushTags = {
+    val lastTag = (cmd("for-each-ref", "--count=1", "--sort=-taggerdate", "--format", "%(tag)", "refs/tags") !!).
+      linesIterator.next()
```

Comment:
This feels a bit fragile - we know what the tag was because we tagged it, so we should use that.  It will require a change to the signature of `pushChanges`, to take the tag (maybe an option of the tag), and it'll also require storing the tag in the state.